tests/Unit/Context/ContextTest.php

```diff
@@ -177,4 +178,19 @@ public function test_attach_and_detach_set_current_ctx(): void
             $scope->detach();
         }
     }
+
+    public function test_debug_scopes_disabled_env_var(): void
+    {
+        \putenv('OTEL_PHP_DEBUG_SCOPES_DISABLED=1');
```

Comment:
I'm worried this might interfere with other tests, since it modifies the environment and doesn't reset it.

Reply:
My bad; it would have interfered with other tests!